### ISSUE TITLE
Would this work together with Real-Time-Voice-Cloning?

### ISSUE THREAD

**Issue by BurguerJohn** [jrgillick/imagined-lyrics#1]
After generating the new lyrics, it would be possible to generate the timestamp as well?
That  way you could clone the voice using
https://github.com/CorentinJ/Real-Time-Voice-Cloning
and stitch it together to the original instrumental.

**Comment by jrgillick** [comment]
Yeah, you could get a rough alignment for the timing of the words, though it wouldn't be perfect. That would be pretty fun :)
